### DISCUSSION TITLE
PLUGIN RACK

### DISCUSSION THREAD

**Discussion by Dumdididum** [apohl79/audiogridder#1293] [Ideas]
How about a plugin rack that will be excepted by AG server that can load all those plugins that can not be loaded into the server directly by itself?

**Comment by emack999** [comment]
There is [BlueCat Patchwork](https://www.bluecataudio.com/Products/Product_PatchWork/) and [DDMF Metaplugin](https://ddmf.eu/metaplugin-chainer-vst-au-rtas-aax-wrapper/) for that kind of plugin wrapper.

**Comment by Dumdididum** [comment]
Thanks, I'll check it out!



*William Fleming*
*FlemingSound*
*(989)400-0140 <http://voice.google.com/calls?a=nc,%2B19894000140>*
***@***.*** ***@***.***>*
